.circleci/config.base.yml

```diff
@@ -51,7 +51,7 @@ install_cli_from_local_registry: &install_cli_from_local_registry
     startLocalRegistry "$(pwd)/.circleci/verdaccio.yaml"
     setNpmRegistryUrlToLocal
     changeNpmGlobalPath
-    npm install -g @aws-amplify/cli-internal@^11.0.0-zzz-beta.0
+    npm install -g @aws-amplify/cli-internal@^11.0.0
```

Comment:
can we just use whatever is the latest available and remove `@^11.0.0`? Saves us from keep updating this file.

Reply:
That matches all `11.xx.yy` versions, so you'll get latest until we ship major version. See also comments above https://github.com/aws-amplify/amplify-category-api/pull/1339#discussion_r1134480879 .

I'm fine either way you and @alharris-at decide.

Reply:
removed.